source/guides/getting-started.md

```diff
@@ -1011,8 +1012,8 @@ Open up `apps/web/templates/books/new.html.erb`:
   <div class="errors">
     <h3>There was a problem with your submission</h3>
     <ul>
-      <% params.errors.each do |error| %>
-        <li><%= error.attribute_name %> is required</li>
+      <% params.errors(full: true).each do |error| %>
```

Comment:
@bruz Thank you for catching this mistake. That error is available for the raw `Dry::Validation::Result`, but it wasn't for `Params`.

I introduced a new method `Params#error_messages` and updated the documentation accordingly. https://github.com/hanami/controller/pull/162